client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -26,14 +23,27 @@ object FlightComponents {
     if (flightWithSplits.apiFlight.Origin.isDomesticOrCta)
       "pax-no-splits"
     else flightWithSplits.bestSplits match {
-      case Some(Splits(_, SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages, _, _)) => "pax-api"
+      case Some(Splits(_, SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages, _, _)) if isLiveAndApiPaxCountWithinThreshold(flightWithSplits) => "pax-api"
       case Some(Splits(_, SplitSources.PredictedSplitsWithHistoricalEGateAndFTPercentages, _, _)) => "pax-predicted"
       case Some(Splits(_, SplitSources.Historical, _, _)) => "pax-portfeed"
       case _ => "pax-unknown"
     }
 
+
+  def isLiveAndApiPaxCountWithinThreshold(flightWithSplits: ApiFlightWithSplits, threshold: Double = 0.05) : Boolean = {
```

Comment:
can we put this function on the ApiFlightsWithSplits class as I think it will be useful elsewhere - perhaps we could name it `hasValidApiSplits` and we can just hard code the threshold in the function, so no parameters needed.

Reply:
this could also do with some unit tests, which would be easy to do if you put this function on the ApiFlightWithSplits class.

Reply:
will hasApiWithinThreshold make more sense here ?

Reply:
happy with that name as well!